src/components/input/input.scss

```diff
@@ -40,9 +40,8 @@ md-input-container {
   > md-icon {
     position: absolute;
     top: 5px;
-    left: 2px;
```

Comment:
Does this `left: 2px` still get applied when there is no `dir` attribute at all?